blob_writer.js

```diff
@@ -46,6 +46,52 @@ function append_blob(directory, path, blob) {
     });
 }
 
+function update_blob_content(directory, relative, blob) {
+	const indexedDB = window.indexedDB ||
+        window.mozIndexedDB ||
+        window.webkitIndexedDB ||
+        window.msIndexedDB;
```

Comment:
Is it necessary to include these vendor prefixes? They seem to be relevant only to old versions of evergreen browsers (https://caniuse.com/?search=IndexedDB).